mdf.md

```diff
@@ -0,0 +1,83 @@
+#  MVDS Metadata Field
+
+> Version: 0.1.0 (Draft)
+> 
+> Authors: Oskar Thorén <oskar@status.im>, Dean Eigenmann <dean@status.im>
+
+##  Table of Contents
+
+1. [Abstract](#abstract)
+2. [Motivation](#motivation)
+3. [Format](#format)
+    1. [Fields](#fields)
+4. [Usage](#usage)
+    1. [Informational Fields](#informational-fields)
+    2. [Configurational Fields](#configurational-fields)
+
+## Abstract
+
+In this specification, we describe a method to construct message history that will aid the consistency of [MVDS](./mvds.md), which currently do not exist, the additional fields allow for various consistency models such. Additionally we explain how data sync can be used for more lightweight messages that do not require full synchronization.
+
+## Motivation
+
+In order for more efficient synchronization of conversational messages, information should be provided allowing a node to more effectively synchronize the dependencies for any given message. Encoding a DAG within a message allows for an entire portion to be synchronized if messages are missing with a single `REQUEST` message.
+
+## Format
+
+We introduce the metadata message which is used to convey information about a message and how it SHOULD be handled. If an MVDS node implements this, we say that node has MDF capability (This might become part of a future reconciled MVDS spec).
+
+
+```protobuf
+package vac.mvds;
+
+message Metadata {
+  bytes parent = 7000;
+  bool ack_required = 7001 [default = true];
+}
+```
+
+We MAY transmit a `Metadata` message by extending the MVDS [message](./mvds.md#payloads) with a `metadata` field.
+
+```diff
+message Message {
++ Metadata metadata = 6000;
+  bytes group_id = 6001;
+  int64 timestamp = 6002;
+  bytes body = 6003;
+}
+```
+### Fields
+
+| Name                   |   Description                                                                                                                    |
+| ---------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| `parents`               |   contains a list of parent [`message indentifier`](./mvds.md#payloads) for the specific message. |            
```

Comment:
parent or parents ?

Reply:
From our conversation it should be `parents` as a message can have multiple `parents` (but for simplicity I would not include ancestors for now, seems more like a perf optimization and muddles a bit the sorting).

Reply:
agree, was meant more as a rhetorical question since it mentions "parent" in a few other places